task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +112,19 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
```

Comment:
there are 2 exit conditions. You need only 1
![image](https://user-images.githubusercontent.com/7272208/88081141-a73cec80-cb88-11ea-8b45-7a53821886f7.png)